test/RestSharp.InteractiveTests/RestSharp.InteractiveTests.csproj

```diff
@@ -8,4 +8,7 @@
         <ProjectReference Include="$(RepoRoot)\src\RestSharp\RestSharp.csproj" />
         <ProjectReference Include="..\RestSharp.Tests.Shared\RestSharp.Tests.Shared.csproj" />
     </ItemGroup>
+    <ItemGroup>
+      <PackageReference Include="xunit.assert" Version="2.4.2" />
```

Comment:
This project doesn't change, why did you add a reference?

Reply:
Think this happed during checking, why the failing tests not working, wanted to find the reason for this.